libs/cardano-ledger-core/src/Cardano/Ledger/Orphans.hs

```diff
@@ -60,6 +66,12 @@ instance NoThunks WC.XSignature where
 instance SignableRepresentation (Hash.Hash a b) where
   getSignableRepresentation = Hash.hashToBytes
 
+-- | TODO: We should upstream instance
+-- HasResolution p => NoThunks (Fixed p) into the nothunks package.
+deriving anyclass instance NoThunks Micro
+
+deriving instance Generic Micro
```

Comment:
I am worried that it can lead to duplicate instance somewhere down stream. I don't think we need this either

Reply:
This is necessary due to the `NoThunks` derivation. GHC complains without it. Is there a better place to put it may be? Or some other way?
```haskell
deriving newtype instance NoThunks (Fixed p)
deriving stock instance Generic (Fixed p)
```
...seems to work out.

Reply:
Sorry, I'm wrong here, by keeping the `DeriveGeneric`. Removing it and just keeping `newtype` with `GeneralizedNewtypeDeriving` did the trick. Thanks a ton! :bow: :heart_eyes: :spiral_notepad: :man_student: :exploding_head: